contracts/interchain-token-service/src/msg.rs

```diff
@@ -95,4 +95,8 @@ pub enum QueryMsg {
     /// Query the configuration parameters for a token
     #[returns(Option<TokenConfig>)]
     TokenConfig { token_id: TokenId },
+
+    /// Query the state of contract (enabled/disabled)
+    #[returns(bool)]
+    IsContractEnabled,
```

Comment:
```suggestion
    IsEnabled,
```
this is how we name the query at the router, should be consistent